apps/docs/pages/guides/platform/multi-factor-authentication.mdx

```diff
@@ -0,0 +1,42 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+import { IconCheck } from 'ui'
+
+export const meta = {
+  title: 'Multi-factor Authentication',
+  description: 'Enable multi-factor authentication (MFA) to keep your account secure.',
+  subtitle: 'Enable multi-factor authentication (MFA) to keep your account secure.',
+}
+
+Multi-factor authentication (MFA) adds an additional layer of security to your organization and prevents unauthorized access to your user account. MFA improves the security of your account by requiring a second factor to verify your user identity. Supabase allows users to enable MFA on their account and set it as a requirement for subsequent logins.
```

Comment:
Given all the docs feedback for SSO, can you add an admonition here something like:

> This guide is for managing MFA on the Supabase dashboard. For adding MFA capabilities to your Supabase project go to ...

And the reverse on the MFA guide for projects too.